dvc/repo/experiments/queue/celery.py

```diff
@@ -193,17 +192,24 @@ def _iter_processed(self) -> Generator[_MessageEntry, None, None]:
             entry_dict = kwargs.get("entry_dict", args[0])
             yield _MessageEntry(msg, QueueEntry.from_dict(entry_dict))
 
-    def _iter_active_tasks(self) -> Generator[_TaskEntry, None, None]:
+    @cached_property
```

Comment:
This should not be a cached property. What we probably want here is a method like
`_get_running_task_ids() -> Set[str]` that will return the set of tasks at the time we called it, and then subsequent calls would continue to return the proper updated set of task IDs